src/math.rs

```diff
@@ -42,14 +42,15 @@ where
     clamped
 }
 
-pub trait PointCulling<S>: objekt::Clone + Debug + Sync + Send
+pub trait PointCulling<S>: Debug + Sync + Send
 where
     S: BaseFloat + Sync + Send,
 {
     fn contains(&self, point: &Point3<S>) -> bool;
     // TODO(catevita): return Relation
     fn intersects_aabb3(&self, aabb: &Aabb3<S>) -> bool;
     fn transformed(&self, global_from_query: &Isometry3<S>) -> Box<dyn PointCulling<S>>;
+    fn corners(&self) -> [Point3<S>; 8];
```

Comment:
Wouldn't this be a little limiting in the future to have this in the trait?

Reply:
I currently think of culling as a bounding volume, which always has corners. I was even thinking about removing `AllPoints` from it, but it would have been too much effort in this PR.
If we need more than a bounding volume, e.g. sphere in the future, we can still take it out of the trait.

Reply:
It also nicely integrates with the changes in the s2 cells mod below...